### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by andgarzonmal**
- [ ]  Use React documentation.
- [ ]  Use React components.
- [ ]  Use React props.
- [ ] Use React Router.
- [ ]  Connect React and Redux.
- [ ]  Handle events in a React app.
- [ ]  Write integration tests with a React testing library.
- [ ]  Use styles in a React app.
- [ ]  Use React life cycle methods.
- [ ]  Apply React best practices and language style guides in code.
- [ ]  Use store, actions and reducers in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+- GitHub: [@githubhandle](https://github.com/andgarzonmal)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/andres-garzon-maldonado-951a2a180/
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (keroloussamy) [inline README.md]: - [ ] Kindly add an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @andgarzonmal, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add unit tests for several components of the app and some pure functions in Redux code. And add integration tests using React Testing Library.




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by andgarzonmal** [comment]
Hello  i added integration tests on my code. Those tests are on the app.test.js file where i mock the user interaction from the main page to the other components (size dogs and selected dog) . Also i added unit tests and checked for several components, i tested Main comonent, App component, Large component, medidium component and small component wich are the main components of my app, also if you check for the coverage of my tests you can see there is a coverage of 95%

image

File: README.md (hunk 1 above)
Comment (Whoistolu) [reply README.md]: - Well Resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @andgarzonmal 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**